index.html

```diff
@@ -0,0 +1,178 @@
+    <script src="https://kit.fontawesome.com/e971436820.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>Anime Characters Global Summit</title>
+</head>
+<body>
+    <header class="header shadow">
+      <div class="logo-div me-auto bd-highlight">
+        <a class="navbar-brand" href="#"><img id="logo-img" src="./assets/images/Logo.png" alt="Logo"></a>
+      </div>
+      <nav class="navbar navbar-expand-md navbar-light">
+        <div class="container-fluid">
+          <button class="navbar-toggler collapsed d-flex d-lg-none flex-column justify-content-around" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="toggler-icon top-bar"></span>
+            <span class="toggler-icon middle-bar"></span>
+            <span class="toggler-icon bottom-bar"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav me-auto ms-3 mb-lg-0 ">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="./about.html">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#program">Program</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#partners">Sponsor</a>
+              </li>
+              <li class="nav-item mb-2 zeromargin">
+                <a class="nav-link active" aria-current="page" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <button type="button" class="menu-btn">AC Campaign</button>
+                <hr>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section id="heading">
+      <img class="heading-bg" src="./assets/images/heading-bg-2.png" alt="konoha panoramic view">
+      <div class="heading-content">
+        <h2>"Hello! Anime FAN"</h2>
+        <header><h1>ANIME CHARACTER<br>GLOBAL SUMMIT</h1><h3 class="year">2022</h3></header>
+        <div class="div-box shadow">
+          <p class="secondary-text">A joyful celebration to reunite all fictional characters that we all love in one place.<br>Scheduled for July 25-29 in Konoha.</p>
+        </div>
+          <h3>2022.07.25(MON) ~ 29(FRI)</h3>
+          <p class="desktop-size">@ Hokage Convention Center of Konoha</p>
+      </div>
+      <button
+        type="button"
+        class="btn btn-outline-secondary btn-floating btn-lg"
+        id="btn-back-to-top"
+        >
+        <i class="fas fa-arrow-up"></i>
+      </button>
+    </section>
+    <section id="program" class="program">
```

Comment:
- [ ] In this `section` you are a small issue with the design as you can see in the screenshots below, the program title is overflowing over the program card, Kindly fix this issue 

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/152209251-b4ba1378-2442-4bb9-9ce5-b79aab9029b1.png)  |  ![](https://user-images.githubusercontent.com/80895497/152209237-fd43c29c-9f76-4658-9c79-b45bb6d0ce50.PNG)